modules/Bio/EnsEMBL/Compara/PipeConfig/Families_conf.pm

```diff
@@ -87,14 +95,22 @@ sub default_options {
         'uniprot_rel_url' => 'ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/reldate.txt',
         'uniprot_ftp_url' => 'ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/uniprot_#uniprot_source#_#tax_div#.dat.gz',
 
-        #'blast_params'    => '', # By default C++ binary has composition stats on and -seg masking off
+        'blast_params'    => '', # By default C++ binary has composition stats on and -seg masking off
+
+        # data directories:
+        'warehouse_dir' => '/nfs/production/panda/ensembl/warehouse/compara/production/'.$self->o('ensembl_release').'/Families_'.$self->o('rel_with_suffix'),
```

Comment:
The path to the compara warehouse feels like something that should be in the ENV, IMO

Reply:
Well you're doing a lot more than just "removing the EBI layer" anyway ;)
I think it's a good point. ENV already has `pipeline_dir` and would welcome a `warehouse_dir` built on the same model that would then be available to all pipelines.